mbofui/app/index.html

```diff
@@ -3,14 +3,14 @@
 
 <head>
   <meta charset="utf-8">
-  <title>{{strings.app_name}}
+  <title>Ocellus
```

Comment:
Is this something we want to hardcode? I thought we were going to get this from a properties file if we changed the name or something.